python_on_whales/components/container/cli_wrapper.py

```diff
@@ -1530,6 +1546,8 @@ def run(
         self._add_publish_to_command(full_cmd, publish)
         full_cmd.add_flag("--publish-all", publish_all)
 
+        full_cmd.add_simple_arg("--pull", "never")
```

Comment:
I think we should only add `--pull never` when the user explicitly ask for it, notably because older docker clients don't have this option and it might render python-on-whales incompatible with those. Other than that, all good :)